__tests__/app.test.js

```diff
@@ -98,6 +98,97 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH:200 updates the value of the votes property on the article with the given article_id, and sends the updated article back to the client", () => {
+    const increment = {
+      inc_votes: 1
+    };
+    return request(app)
+      .patch("/api/articles/10")
+      .send(increment)
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article).toMatchObject({
+          article_id: 10,
+          title: "Seven inspirational thought leaders from Manchester UK",
+          topic: "mitch",
+          author: "rogersop",
+          body: "Who are we kidding, there is only one, and it's Mitch!",
+          created_at: "2020-05-14T04:15:00.000Z",
+          votes: 1,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        });
+      });
+  });
+
+  test("PATCH:200 updates the value of the votes property on the article with the given article_id, and sends the updated article back to the client, while ignoring any unnecessary properties on the object", () => {
+    const decrement = {
```

Comment:
This test description starts exactly the same as the above, you can simplify this with just, "should ignore unnecessary properties", these tests build up on each other so there is never a need to repeat test descriptions like this.